spec/diograph-api/connections/create-connections-spec.ts

```diff
@@ -20,7 +20,9 @@ describe("Diograph API .create('connection')", () => {
       expect(res.data.type).toEqual("connections");
       expect(res.data.attributes["from-diory-id"]).toEqual(parseInt(diory1.id));
       expect(res.data.attributes["to-diory-id"]).toEqual(parseInt(diory2.id));
-      done();
+      // DiographApi.delete(res.data.id, "connections").then(res => {
+        done();
+      // })
```

Comment:
DiographApi.delete doesn't support deleting connections yet...
2 connections are polluted to database on each test run.